src/Sarif/Comparers/ComparerHelpers.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.Comparers
+{
+    internal class ComparerHelper
+    {
+        /// <summary>
+        /// Compare 2 object refences. Return value false presents need to
```

Comment:
>Compare [](http://example.com/codeflow?start=12&length=7)

These comment needs editing, for grammatical correctness and clarity.

Reply:
Worked with Courtney got all user facing strings reviewed